examples/counterexamples/test.properties

```diff
@@ -0,0 +1,6 @@
+test1.xml.line=9
+test1.xml.col=14
+test1.xml.rule=cvc-complex-type.2.4.a
```

Comment:
How are these rule names constructed?

Please add a `README.md` explaining how to add new test cases